utensor_cgen/legalizer/tflite.py

```diff
@@ -7,6 +7,7 @@
 
 from .api import Legalizer
 from .base import LegalizerBase
+from .utils import _hotfix_reshape
```

Comment:
May be worth renaming this operation at some stage

Reply:
Ya. We are not sure if this hotfix is a good idea or not.
Let's leave it as is for now.